### ISSUE TITLE
Authorization header bug when using python 3

### ISSUE THREAD

**Issue by trevorandersen** [RDFLib/sparqlwrapper#71]
When using setCredentials('admin', 'admin') with python 3, the authorization header subsequently sent is invalid. It is
> 'Authorization': "Basic b'YWRtaW46YWRtaW4='"

rather than

> 'Authorization': 'Basic YWRtaW46YWRtaW4='

The issue occurs because b64encode returns bytes in python 3. A simple fix is to modify line 510 to add a call to **decode**
```python
# from
request.add_header("Authorization", "Basic %s" % base64.b64encode(credentials.encode('utf-8')))
# to
request.add_header("Authorization", "Basic %s" % base64.b64encode(credentials.encode('utf-8')).decode('utf-8'))

**Comment by trevorandersen** [comment]
Pull request #72

**Comment by wikier** [comment]
I merged the patch from #72. For closing this issue @trevorandersen it'd be cool to have a test over the feature; do you think you cloud provide it?

**Comment by trevorandersen** [comment]
Done. It was a matter of deleting the old test code which explicitly checked for the b' characters which broke authorization, and restoring the simple test which had been commented out. #73

**Comment by wikier** [comment]
True. So I merged pr #73 and ci looks happy https://travis-ci.org/RDFLib/sparqlwrapper/builds/97619101 :smile:
